iocage/ListableResource.py

```diff
@@ -81,7 +84,7 @@ def __iter__(
         self
     ) -> typing.Generator['iocage.Resource.Resource', None, None]:
         """Return an iterator over the child datasets."""
-        if self.namespace is None:
```

Comment:
Perhaps we make this an OR with the below addition?

Reply:
I am not sure if I followed you correctly. Are you suggesting `self.namespace is None or not self.namespace` ?

Reply:
Yep! Stefan probably has a specific use for None.

Reply:
Oh, I was not aware of that. `not self.namespace` should normally cover the general negation scenario but if we have a specific use case, then that needs to be followed through. Let's confirm this ? @gronke could you confirm this please ?